app/pb_kits/playbook/pb_checkbox/_checkbox.scss

```diff
@@ -0,0 +1,66 @@
+@import "../tokens/colors";
+$transition: 0.2s cubic-bezier(0.95, 0.05, 0.795, 0.035);
```

Comment:
Should this be added to the transitions as a token? - [Transition Tokens](https://github.com/powerhome/playbook/blob/master/app/pb_kits/playbook/tokens/_transition.scss)